lib/ansible/modules/yum_repository.py

```diff
@@ -674,7 +674,8 @@ def main():
 
     # Rename "name" and "description" to ensure correct key sorting
     module.params['repoid'] = module.params['name']
-    module.params['name'] = module.params['description']
+    module.params['name'] = module.params['name']
+    module.params['description'] = module.params['description']
```

Comment:
This fix is does not seem to make sense. Changing line 677 does make a change, but the added two other lines are unnecessary.